README.md

```diff
@@ -2,11 +2,12 @@
     <img src="https://raw.githubusercontent.com/ManimCommunity/manim/master/logo/cropped.png" />
```

Comment:
Can you link to `https://www.manim.community`?

Reply:
Done. 👍